### PR TITLE
chore: add github ci

### PR DESCRIPTION
**Pull request by AnguIar**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Chore            | ✔                                                                       |

Add CI to boilerplate

### CODE REVIEW

File: .github/workflows/build_and_push.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+on:
+  workflow_dispatch:
```
Comment (syncush) [inline .github/workflows/build_and_push.yaml:2]: if i use dispatch, will it use the git sha instead of tag?

**Review by syncush — COMMENTED** [review]
isnt there an action specifically for ACR? why the hustle with docker actions?

File: .github/workflows/build_and_push.yaml (hunk 1 above)
Comment (AnguIar) [reply .github/workflows/build_and_push.yaml:2]: branch name / tag
it's possible to use commit SHA if we want to (aka https://stackoverflow.com/questions/65895649/how-to-pre-fill-an-input-field-in-a-workflow-dispatch-github-action-with-github)

Comment (syncush) [reply .github/workflows/build_and_push.yaml:2]: it is suffiencient

**Pull request merged**